### PR TITLE
Move sandbox-classic test lib and IT tests to Sandbox-on-x

### PR DESCRIPTION
**Pull request by tudor-da**
This PR:
* Moves the `SandboxFixture` from `sandbox-classic` to `sandbox-on-x`, since it is using the Sandbox-on-X internals. This allows further down the road replacement of the sandbox-next fixtures.
* Moves the integration tests from `sandbox-classic` to `sandbox-on-x`
* Moves `SandboxServer` (the sandbox-classic server) and its utils to `sandbox-on-x:sandbox-classic-server` since they are needed by the integration tests ^.  

After these changes, `sandbox-classic` keeps only its shell and conformance tests.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox-classic/BUILD.bazel [hunk 1]
```diff
@@ -11,241 +11,114 @@ load("//ledger/ledger-api-test-tool:conformance.bzl", "server_conformance_test")
...
+    "//libs-scala/ports",
+    "//libs-scala/resources",
+    "@maven//:ch_qos_logback_logback_classic",
+]
+
+# TODO sandbox-classic cleanup: De-alias
 alias(
-    name = "sandbox-classic",
-    actual = "sandbox-classic-ce",
+    name = "sandbox-classic-scala-tests-lib",
```
Comment (tudor-da) [inline ledger/sandbox-classic/BUILD.bazel:35]: Aliased this here in order to not introduce more changes to its dependencies in this PR.

Comment (stefanobaghino-da) [reply ledger/sandbox-classic/BUILD.bazel:41]: Should we wait for those changes before approving?

File: ledger/sandbox-classic/BUILD.bazel [hunk 2]
```diff
@@ -11,242 +11,120 @@ load("//ledger/ledger-api-test-tool:conformance.bzl", "server_conformance_test")
...
-]
+# TODO sandbox-classic cleanup: De-alias
+alias(
+    name = "sandbox-classic-scala-tests-lib",
+    actual = "//ledger/sandbox-on-x:sandbox-on-x-test-lib",
+    visibility = ["//visibility:public"],
+)
+
+da_scala_library(
+    name = "sandbox-classic-ce",
```
Comment (stefanobaghino-da) [inline ledger/sandbox-classic/BUILD.bazel:47]: At this point the `-ce` suffix can probably be removed and the alias above removed.

**Review by stefanobaghino-da — COMMENTED** [review]
LGTM but I couldn't get from one outstanding TODO whether the intention is to merge this or finish working on that before merging.

File: ledger/sandbox-classic/BUILD.bazel (hunk 1 above)
Comment (tudor-da) [reply ledger/sandbox-classic/BUILD.bazel:35]: No, need. Left them for a subsequent PR (to limit the scope of the changes here)

File: ledger/sandbox-classic/BUILD.bazel (hunk 2 above)
Comment (tudor-da) [reply ledger/sandbox-classic/BUILD.bazel]: done 👍

**Review by stefanobaghino-da — APPROVED** [review]

File: ledger/sandbox-on-x/src/classic/scala/com/daml/platform/sandbox/ConfigConverter.scala [hunk 3]
```diff
@@ -1,24 +1,16 @@
...
-  Mode,
-  ParticipantConfig,
-  ParticipantIndexerConfig,
-  ParticipantRunMode,
-}
-import ledger.sandbox.{BridgeConfig, BridgeConfigProvider}
-import lf.language.LanguageVersion
-import platform.common.LedgerIdMode
-import platform.sandbox.config.SandboxConfig.EngineMode
+package com.daml.platform.sandbox
```
Comment (mziolekda) [inline ledger/sandbox-on-x/src/classic/scala/com/daml/platform/sandbox/ConfigConverter.scala:4]: Why not `com.daml.ledger.sandbox`?

Comment (mziolekda) [reply ledger/sandbox-on-x/src/classic/scala/com/daml/platform/sandbox/ConfigConverter.scala:4]: Same pertains to all subsequent files

File: release/artifacts.yaml [hunk 4]
```diff
@@ -159,7 +159,9 @@
   type: jar-scala
 - target: //ledger/sandbox-common:sandbox-common-ce
   type: jar-scala
-- target: //ledger/sandbox-classic:sandbox-classic-ce
+- target: //ledger/sandbox-classic:sandbox-classic
```
Comment (mziolekda) [inline release/artifacts.yaml:162]: I'd love to see this one go completely from the published artifacts.
It is probably better to do it in a separate PR though.

**Review by mziolekda — APPROVED** [review]
I'd prefer to get rid of the `com.damll.platform.sandbox` inside the `sandbox-on-x` project. It feels like a foreign body there. I could settle for a subsequent PR though.
